src/main/java/com/direwolf20/buildinggadgets/common/concurrent/CopyScheduler.java

```diff
@@ -0,0 +1,57 @@
+package com.direwolf20.buildinggadgets.common.concurrent;
+
+import com.direwolf20.buildinggadgets.common.building.BlockData;
+import com.direwolf20.buildinggadgets.common.building.PlacementTarget;
+import com.direwolf20.buildinggadgets.common.building.Region;
+import com.direwolf20.buildinggadgets.common.building.view.IBuildContext;
+import com.direwolf20.buildinggadgets.common.building.view.IBuildView;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.Objects;
+import java.util.Spliterator;
+import java.util.function.BiConsumer;
+
+public final class CopyScheduler extends SteppedScheduler {
```

Comment:
Yes, why shouldn't it be?!?

Reply:
(I presume that's what you're asking: Copying was never off-thread, because it needs to touch the world. This is a scheduler to allow for large copies to be performed in multiple segments as per config-option)

Reply:
Nope, was just hoping all that had been removed. I'm still in two minds on what kinda scale we should be supporting.